README.md

```diff
@@ -48,14 +48,14 @@ Identify Stakeholders and Validate Interest
 - Identify market segments and stakeholders
 - Understand Who cares about Open Source Value (and why)
 - Specify and prioritize Key Performance Indicators
-- Calculate potential aggregate economic value
+- Calculate the potential aggregate economic value
 
 Grow hands-on community
 - CHAOSS Members: Consultants, researchers, grant writers
 - Project Stakeholders: Sponsors, maintainers, contributors, consumers
 
 Build reusable assets
-- Repeatable methodology for discovering value
+- Repeatable methodology for discovering the value
```

Comment:
I disagree with the edit in line 58. "the" is not needed here since we are not referring to a single and specified value but any kind of value.

Reply:
:+1: